### PR TITLE
updated api doc and api for rating_create 

### PR DESCRIPTION
**Pull request by daminichopra**
Fixes #
Fix for [Issue](https://github.com/ckan/ckan/issues/5558)
### Proposed fixes:
Removed deprecated code for rating_create Api and updated docs for same.

### Features:

- [X] includes tests covering changes
- [X ] includes updated documentation
- [X ] includes user-visible changes
- [X ] includes API changes
- [X] includes bugfix for possible backport

Please [X] all the boxes above that apply

### CODE REVIEW

**Comment by amercader** [comment]
Thanks for this @daminichopra , is this still a draft?

**Comment by jqnatividad** [comment]
What about [model/package.py](https://github.com/ckan/ckan/blob/master/ckan/model/package.py)?

There are still a lot of references to ratings there...

**Comment by daminichopra** [comment]
> What about [model/package.py](https://github.com/ckan/ckan/blob/master/ckan/model/package.py)?
> 
> There are still a lot of references to ratings there...

@jqnatividad , I have updated the PR. Thank you

**Comment by daminichopra** [comment]
> Thanks for this @daminichopra , is this still a draft?

Hi @amercader , I have removed it from draft. Thank you

**Comment by daminichopra** [comment]
Hi @amercader , Please merge and close this issue if it looks good.Thank you :)

File: doc/api/index.rst [hunk 1]
```diff
@@ -433,6 +433,8 @@ ckan.logic.action.create
 
 .. automodule:: ckan.logic.action.create
    :members:
+.. note::
+   Please refer link for `rating_create <https://github.com/6aika/ckanext-rating>`
```
Comment (amercader) [inline doc/api/index.rst:437]: I don't think we need these two lines here

```suggestion
```

**Comment by Gauravp-NEC** [comment]
@amercader , This PR is no longer needed. The issue for which this PR was raised is already closed by another PR. You can close this PR. Thanks

**Comment by Gauravp-NEC** [comment]
> @Gauravp-NEC what other PR? Please provide a link when mentioning other PRs

@amercader , Please find the link of other PR https://github.com/ckan/ckan/pull/6311. Issue https://github.com/ckan/ckan/issues/5558

**Comment by amercader** [comment]
Closed in favour of #6311

**Pull request closed without merging**